orchagent/orch.cpp

```diff
@@ -866,6 +879,15 @@ void Orch2::doTask(Consumer &consumer)
     while (it != consumer.m_toSync.end())
     {
         bool erase_from_queue = true;
+        bool sync_mode = false;
+        ReturnCode rc;
+        for (const auto &kv : kfvFieldsValues(it->second))
+        {
+            if (to_upper(fvField(kv)) == "SYNC_MODE" && to_upper(fvValue(kv)) == "TRUE")
```

Comment:
@prsunny
Hi, Prince. Just want to confirm. Are you suggesting a global command line flag for the response path?
If we do that, orchagent will do the same handling on all requests.
(Currently, we only have response path on P4Orch and VrfOrch.)